pkgs/development/python-modules/tempita/default.nix

```diff
@@ -1,20 +1,18 @@
-{ lib, buildPythonPackage, fetchFromGitHub, nose }:
+{ lib, buildPythonPackage, fetchurl, nose }:
 
 buildPythonPackage {
-  version = "0.5.3-2016-09-28";
+  version = "0.5.3";
   pname = "tempita";
 
-  src = fetchFromGitHub {
-    owner = "gjhiggins";
-    repo = "tempita";
-    rev = "47414a7c6e46a9a9afe78f0bce2ea299fa84d10";
-    sha256 = "0f33jjjs5rvp7ar2j6ggyfykcrsrn04jaqcq71qfvycf6b7nw3rn";
+  src = fetchurl {
```

Comment:
Why not `fetchFromBitbucket`?

Reply:
```
fetchFromBitbucket {
  owner = "ianb";
  repo = pname;
  rev = "97392d008cc8";
  sha256 = "0nxnkxjvfyxygmws2zxql590mwqsqd1rnhy80m9nbpdh81p7vh9a";
};
```

Reply:
Will do it tomorrow, thanks.

Reply:
> Will do it tomorrow, thanks.

Tomorrow was long ago. @jpgu-epam tempita is still broken on master, is there any chance you could reach to it?